### DISCUSSION TITLE
Add ability to report score on lazer leaderboard.

### DISCUSSION THREAD

**Discussion by Coppertine** [ppy/osu#20071] [Ideas]
Looking through different leaderboards for high star raiting ranked maps. I noticed that there are some scores that are extremely cheated.
One of those examples is GDceherrios' score on UNDEAD CORPORATION - Blow [A bit of luck!] (https://osu.ppy.sh/beatmapsets/1019640)
Where the score itself has 1330 max combo, that being 1536 300's and 1 miss in the entire map.
Below is a screenshot:
(Running version: 2022.901.0)
![osu_2022-09-01_22-26-31](https://user-images.githubusercontent.com/37494321/187915033-859727da-c0ff-4c09-83d7-eb0404d57c06.png)

**Comment by Doksha** [comment]
This is due to a bug where you could submit unranked scores (ie. delete every hit object and replace it with a buzz slider or something) and it would submit to the online leaderboard (It has been fixed since). Its not 'cheated' per se, but old scores like that should be fixed eventually. I would guess the lazer servers will eventually take on the task of automatically checking replays, but that likely wont be for a while

**Reply by Coppertine** [reply]
That was of a different case. This case, 1,533 objects were hit with a Great judgement as well as 1 miss. This is almost the max combo of this spesific map.
 Even if this score was recalculated, it would lead to around the same score, or even higher.
 Noting that this map itself is 230bpm with extreme stream and jump sections, I have a major feeling this is a cheated play.
